src/main/scala/io/iohk/ethereum/domain/BlockHeader.scala

```diff
@@ -102,24 +128,42 @@ object BlockHeader {
   }
 
   implicit class BlockheaderEncodableDec(val rlpEncodeable: RLPEncodeable) extends AnyVal {
+    private def encodeOptOut(encodedOptOut: RLPEncodeable): Option[Boolean] = {
```

Comment:
:+1: 

Isn't this decodeOptOut?

Reply:
Yes, `decode`. How is this related to checkpointing? Is that it was simply missed on a previous PR?

Reply:
this is encoder/decoder for a header with checkpoint field which is compatible with the old blocks.